src/routes/Products/components/BasicInfo/index.jsx

```diff
@@ -177,19 +177,22 @@ const BasicInfo = ({
       dispatch(triggerAutoSave(true));
     }
 
+    console.log(basicInfo, "basicInfo");
     if (!!basicInfo?.projectTitle?.value?.length) {
       setFormData(basicInfo);
     }
 
     setFirstMounted(false);
 
     return () => {
-      dispatch(triggerAutoSave(true));
+      console.log(formData, "form data");
+      // dispatch(triggerAutoSave(true));
```

Comment:
@brooketopcoder Just want to know why the autoSave is triggered on unmount of the component? This introduced (https://topcoder.atlassian.net/browse/PROD-2141) this bug in the code when opening the last saved draft challenge, the form becomes empty.

Reply:
This happens because somehow the autosave in this line overrides the existing values with the default state.

Reply:
@brooketopcoder can you please clarify here?

Reply:
@brooketopcoder probably you missed this thread :) can you please clarify this?